electron-app/src/store/balances/getters.ts

```diff
@@ -112,6 +114,33 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
     }, Zero);
   },
 
+  manualBalances: (state: BalanceState) => {
```

Comment:
Since we just access the manualBalances state property, it doesn't make sense to use a getter. We can do a map state directly in the component instead.

Reply:
Could you explain this a bit more? So I just delete this getter? What changes would I then make in `ManualBalancesList.vue`?